### ISSUE TITLE
Feature Request for fetching the task information #83770846

### ISSUE THREAD

**Issue by asifhuawei** [cloudfoundry-incubator/lattice#117]
I was going through Lattice project tracker and found a story [#83770846](https://www.pivotaltracker.com/story/show/83770846) which needs to fetch the task information using task guid.

I have implemented the above story in the following way

```
root@ubuntu:~# ltc help tasks
NAME:
   tasks - Fetches information of tasks running on lattice

USAGE:
   command tasks [arguments...]

DESCRIPTION:
   ltc tasks TASK_GUID

root@ubuntu:~# ltc tasks task-guid-1
Displaying the information of task task-guid-1
Task Guid:              task-guid-1
Cell ID:
Status:                 PENDING

root@ubuntu:~# ltc tasks task-guid-1
Displaying the information of task task-guid-1
Task Guid:              task-guid-1
Cell ID:                lattice-cell-01
Status:                 RUNNING

root@ubuntu:~# ltc tasks task-guid-1
Displaying the information of task task-guid-1
Task Guid:              task-guid-1
Cell ID:                lattice-cell-01
Status:                 COMPLETED
Failiure Reason:        failed to initialize container

root@ubuntu:~# ltc tasks task-guid-123757652735
Displaying the information of task task-guid-123757652735
Error while fetching the task by GUID : task with guid 'task-guid-123757652735' not found
root@ubuntu:~# ltc tasks
Please enter the task guid

```

![realtasks](https://cloud.githubusercontent.com/assets/9736066/7491407/d407cb68-f406-11e4-8f11-7556b8376ab9.PNG)

@davidwadden 
If this implementation fulfills the need of the above story then I am ready to raise a PR for this or else I am ready to make changes according to feedback.

Waiting for you feedback.....

**Comment by menicosia** [comment]
Hi Asif,

I know it can be hard for people who are not used to Tracker to understand, but what you have done is work on a story that has been stored in our 'Icebox.' In Tracker, the way work is organized is based on stories that are actionable and have been prioritized. Those are the only stories we'll typically work on, and they're found in the Current and Backlog columns.

Stories that are not prioritized (for any number of reasons) are kept in the Icebox. Those stories are a grab-bag of various stories, from feature requests, partially developed ideas, or the result of brainstorming. The Icebox is a repository for all stories that are under consideration, and may possibly undergo considerable transformation or be discarded entirely. Stories in the Icebox may never be prioritized, in fact you'll see at the bottom of the Icebox is a section called, 'Graveyard.' Stories are stored there specifically to record that an idea has been considered and discarded.

The Icebox is not always maintained in the same disciplined manner of the Backlog.

In the case of Tasks, they were a partially-fleshed out idea that are not likely to be prioritized. We've decided to go in a different direction, but the Tracker hasn't been updated to reflect that. So unfortunately, we won't be able to accept your implementation. I know this is frustrating, and I'm sorry. We're really delighted to have your help these past several weeks.

I do have an alternate system to propose:
Several days ago, @davidwadden went through and tagged stories both in the Backlog and the Icebox as 'community.' These are stories that we felt are particularly well-suited to community contribution, and feel would be excellent candidates for people such as yourself to pull and work on. In the reality that the Icebox will never be a perfectly organized prediction of all the work we'll do, the 'community' tag is a compromise that there are a variety of features we're confident will be welcomed.

I'm going to close this Issue, but of course you should always feel welcome to re-open it if you find that there's value that would be lost.

--
  Marco Nicosia
  Product Manager
  Pivotal Software, Inc.

**Comment by davidwadden** [comment]
You know what, if you're still interested, and I apologize ahead of time for the short notice, but we decided to pull those tasks stories into this release after all to help us out with [condenser](https://www.pivotaltracker.com/epic/show/1820828).  If you're interested and able to deliver something for us to review by Tuesday or Wednesday this week (Monday is a holiday for us), we'd consider either either of the following stories.  As you can see on our [Tracker](https://www.pivotaltracker.com/n/projects/1183596), we're trying to cut v0.2.5 before the end of next week and want to get the tasks epic in.

[**#83770848**](https://www.pivotaltracker.com/story/show/83770848) - ltc should be able to cancel tasks/remove completed tasks
[**#84348820**](https://www.pivotaltracker.com/story/show/84348820) - `ltc list` should also list tasks

If you're interested, go ahead and open new Github issue(s) for the above stories to if you have any questions.  Please also take a look at our recently implemented `ltc tasks` ([**#83770846**](https://www.pivotaltracker.com/story/show/83770846)) and `ltc submit-task` ([**#83770844**](https://www.pivotaltracker.com/story/show/83770844)), and feel free to submit any improvements you may have for those.

Thanks again for all your support! 
David

/cc @asifhuawei @ShwethaKumbla @dhilipkumars @savitaashture

**Comment by asifhuawei** [comment]
Hi @davidwadden thanks for considering us to do this task, we will be happy to work on that as we have recently explored a lot on task related things and we will try to raise a PR for [#83770848](https://www.pivotaltracker.com/story/show/83770848) [#84348820](https://www.pivotaltracker.com/story/show/84348820) by Tuesday.

Thanks 
Asif
